src/test/helpers/db.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from "react";
+import { act } from "@testing-library/react-native";
+import { DbContext } from "../../context/db";
+
+export const mockSubscribe = jest.fn();
+export const mockInsert = jest.fn();
+
+const documents = {
+  find: () => documents,
+  findOne: () => documents,
+  sort: () => documents,
+  insert: mockInsert,
+  $: {
+    subscribe: mockSubscribe
+  }
+};
+const mockDb: any = {
```

Comment:
Do we need the `any` here?

Reply:
Yep, I don/t want to go through the trouble to make a class with the methods that conforms to the RxDB class, so I foresee this helper to evolve as it goes when more methods are used.